test/functional/package/list_test.rb

```diff
@@ -72,23 +72,15 @@ module HammerCLIKatello
     describe 'environment options' do
       it 'may be specified environment name' do
         org_id = 2
-        env_name = "dev"
+        env_id = 3
         expected_id = 6
-        expect_lifecycle_environment_search(org_id, env_name, expected_id)
+        expect_lifecycle_environment_search(org_id, env_id, expected_id)
```

Comment:
If it searches for environment why you drop the name here? I assume that it does `index` call with the name to grab the id.

Reply:
ok fair point I will switch it back